File: iniciante/c99/1012/1012.java
//QUESTÃO 1012 - Área

import java.io.IOException;
import java.util.Scanner;
 
public class Main {
	public static void main(String[] args) throws IOException {
   
	   	Scanner sc = new Scanner(System.in);
			            
	        double a = sc.nextDouble();
 	        double b = sc.nextDouble();
 	        double c = sc.nextDouble();
						            
		double PI = 3.14159;
							            
	        double triangulo = (a * c) / 2;	
		double circulo = (Math.pow(c, 2) * PI);
	        double trapezio = ((a + b) * c) / 2;	
		double quadrado = Math.pow(b, 2);
		double retangulo = a * b;
												            

		String saidaTriangulo = String.format("TRIANGULO: %.3f", triangulo);

		String saidaCirculo = String.format("CIRCULO: %.3f", circulo);

		String saidaTrapezio = String.format("TRAPEZIO: %.3f", trapezio);

		String saidaQuadrado = String.format("QUADRADO: %.3f", quadrado);

		String saidaRetangulo = String.format("RETANGULO: %.3f", retangulo);


		
		System.out.println(saidaTriangulo);
		System.out.println(saidaCirculo);
		System.out.println(saidaTrapezio);
		System.out.println(saidaQuadrado);
		System.out.println(saidaRetangulo);													        }
}

File: iniciante/c99/1012/1012.c
#include <stdio.h>
#include <math.h>

int main() {

	double a, b, c;
	const double PI = 3.14159;

	scanf("%lf", &a);
	scanf("%lf", &b);
	scanf("%lf", &c);

	double areaTriangulo = a * c / 2;
	double areaCirculo = PI * pow(c, 2);
	double areaTrapezio = (a + b) * c / 2;
	double areaQuadrado = pow(b, 2);
	double areaRetangulo = a * b;

	printf("TRIANGULO: %.3lf\n", areaTriangulo);
	printf("CIRCULO: %.3lf\n", areaCirculo);
	printf("TRAPEZIO: %.3lf\n", areaTrapezio);
	printf("QUADRADO: %.3lf\n", areaQuadrado);
	printf("RETANGULO: %.3lf\n", areaRetangulo);

	return 0;

}
